webapp/package.json

```diff
@@ -30,7 +30,7 @@
     "@angular/platform-browser-dynamic": "18.2.1",
     "@angular/router": "18.2.1",
     "@devolutions/icons": "4.0.8",
-    "@devolutions/iron-remote-desktop": "^0.7.0",
+    "@devolutions/iron-remote-desktop": "../../ironrdp/web-client/iron-remote-desktop/dist",
```

Comment:
Waiting for https://github.com/Devolutions/IronRDP/pull/944 to be merged and released